### PR TITLE
fixed search forms using standard from angular document

### PR DESCRIPTION
**Pull request by Asare12**

### CODE REVIEW

File: KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.component.ts [hunk 1]
```diff
@@ -16,8 +17,37 @@ export class HomeComponent implements OnInit {
...
+    occupation: [""],
+    registrarName: [""],
+    gender: [this.gender],
+    age: [""],
+    religion: [this.religion],
+    deathDate: [""],
+    burialDate: [""],
+  });
+
+  searchGraveForm = this.fb.group({
```
Comment (abdulfs2018) [inline KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.component.ts:36]: cant have 2 different form groups. move the form control stuff into a single form group.

File: KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.component.ts [hunk 2]
```diff
@@ -16,8 +17,37 @@ export class HomeComponent implements OnInit {
   readonly style = "style";
   readonly SEARCH_KEY = "local_search";
 
-  constructor( private dspcDataProvider: DSPCExplorerDataProvider, private localStorageService: DSPCExplorerLocalStorageProvider) {
-    this.display = this.localStorageService.getFromLocalStorage(this.SEARCH_KEY) !== undefined;
+  gender: string[] = ["Male", "Female"];
+  religion: string[] = ["Roman Catholic", "Protestant"];
+  graveSize: string[] = ["Single", "Double", "Triple", "Quadruple", "Large"];
```
Comment (abdulfs2018) [inline KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.component.ts:22]: why this? can the forms not hold the values?

File: KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.component.ts [hunk 3]
```diff
@@ -29,19 +59,17 @@ export class HomeComponent implements OnInit {
...
         graveRefences.push(e.graveOwner.graveReferenceCode);
 
         let searchData: SearchResultViewModel = {
           graveownerName: e.graveOwner.graveOwnerName,
           graveRefCode: e.graveOwner.graveReferenceCode,
           buttonText: "View Selected Grave Details",
           graveSize: e.graveOwner.graveSize,
-          imageSource: "grave-headstone-sample.jpg"
-        }
+          imageSource: "grave-headstone-sample.jpg",
```
Comment (abdulfs2018) [inline KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.component.ts:71]: no change needed here.

File: KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.module.ts [hunk 4]
```diff
@@ -1,55 +1,53 @@
-import { CommonModule } from '@angular/common';
-import { NgModule } from '@angular/core';
-import { NgbModule } from '@ng-bootstrap/ng-bootstrap';
-import { RouterModule, Routes } from '@angular/router';
+import { CommonModule } from "@angular/common";
```
Comment (abdulfs2018) [inline KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.module.ts:1]: why does this entire file look like it was replaced ? can you please double check this.. should'nt be in review if only one change

Comment (Asare12) [reply KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.module.ts:1]: It wasn't replaced Common Module just got moved up

Comment (abdulfs2018) [reply KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.module.ts:1]: i dont mean that line.. the entire file itself is showed as green.. like new changes were made or so

Comment (Asare12) [reply KAIS.Interactive.DSPC_EXPLORER.UI/dspc-explorer-ui/src/app/screens/home/home.module.ts:1]: Ahh yeah I just added ReactiveFormsModule

**Pull request merged**